src/features/presence/utils/presenceUtils.js

```diff
@@ -60,4 +64,24 @@ function determineInteractionMessage(interactionType, interactedAt) {
     return message;
 }
 
-export { sortByActivity, getLastActionTimeMS, determineInteractionMessage };
+function convertMillisecondsToUnitAndValue(ms) {
+    const seconds = ms / 1000;
+    const abs = Math.abs(seconds);
```

Comment:
might be redundant to do an absolute check here since there's one when the function is called